<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualDiskWindow</class>
 <widget class="QDialog" name="VisualDiskWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visual Disk - Track Visualization</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBar" name="toolbar">
     <property name="windowTitle">
      <string>toolBar</string>
     </property>
     <addaction name="actionDiskView"/>
     <addaction name="actionGridView"/>
     <addaction name="separator"/>
     <addaction name="actionZoomIn"/>
     <addaction name="actionZoomOut"/>
     <addaction name="separator"/>
     <addaction name="actionExport"/>
     <addaction name="actionRefresh"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupVisualization">
     <property name="title">
      <string>Disk Visualization</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widgetDiskView" native="true">
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>500</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupTrackDetails">
        <property name="title">
         <string>Track Details</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelTrack">
           <property name="text">
            <string>Track:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="labelTrackValue">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelSide">
           <property name="text">
            <string>Side:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelSideValue">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelSectors">
           <property name="text">
            <string>Sectors:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="labelSectorsValue">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelGoodSectors">
           <property name="text">
            <string>Good:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="labelGoodSectorsValue">
           <property name="text">
            <string>-</string>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 255, 0);</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labelBadSectors">
           <property name="text">
            <string>Bad:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="labelBadSectorsValue">
           <property name="text">
            <string>-</string>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="labelRetries">
           <property name="text">
            <string>Retries:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="labelRetriesValue">
           <property name="text">
            <string>-</string>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 0);</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="labelFormat">
           <property name="text">
            <string>Format:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="labelFormatValue">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="labelRPM">
           <property name="text">
            <string>RPM:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="labelRPMValue">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="labelEncoding">
           <property name="text">
            <string>Encoding:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLabel" name="labelEncodingValue">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupLegend">
     <property name="title">
      <string>Legend</string>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelGreenLegend">
        <property name="text">
         <string>● Green = Good</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 255, 0);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelYellowLegend">
        <property name="text">
         <string>● Yellow = Retry</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 0);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelRedLegend">
        <property name="text">
         <string>● Red = Bad</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelBlueLegend">
        <property name="text">
         <string>● Blue = Protection</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 170, 255);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelGrayLegend">
        <property name="text">
         <string>● Gray = Unread</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(128, 128, 128);</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionDiskView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Disk View</string>
   </property>
  </action>
  <action name="actionGridView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Grid View</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>VisualDiskWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1139</x>
     <y>668</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
